ibmcloud/README.md

```diff
@@ -19,12 +19,37 @@ The setup procedure includes the following sub tasks.
 To automate preparation of VPC and VSIs, you need to install terraform and ansible on your client machine. Please follow the the official installation guides.
 
 * [Install Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+
+
+Simply:
```

Comment:
Do we want to mark these instructions as instructions that will only work on (I assume) Ubuntu, or a Linux distro that uses the apt package manager.

I assume the instructions in this README can be ran on any client machine; Windows, Mac OS, any Linux distro, which is why the instructions for installing Terraform and Ansible point at the official guides from hashicorp and the Ansible project

Reply:
Thanks, replace `Simply` to `If you are using Ubuntu linux, you can run follow commands simply`